api/docs/guides/publish.md

```diff
@@ -38,8 +41,11 @@ _Note: [GitHub's documentation provides screenshots for the various steps](https
 1. Navigate to the `openverse-api` "Actions" tab in GitHub: [github.com/WordPress/openverse-api/actions](https://github.com/WordPress/openverse-api/actions).
 2. Select the "Publish Docker images on-demand" workflow.
 3. Click the "Run workflow" button.
-4. Select the target branch to build (this can also be `main`).
-5. For the `image` parameter, specify either `api` or `ingestion_server`.
-6. Click the green "Run workflow" button.
+   1. (Optional) change the branch _that the workflow is running from_ (this does not change anything about the target commit)
+   2. For the `image` parameter, specify either `api` or `ingestion_server`.
+   3. For the `commit` parameter, specify the commit that the built Docker image artifact should be pulled from.
+5. Click the green "Run workflow" button.
 
 The newest run should appear in the Action list, and its status can be tracked from there.
+
+![Example Screenshot](/_static/publish_action_example.png)
```

Comment:
Nice! Can we remove the [`TODO`](https://github.com/WordPress/openverse-api/blob/main/api/docs/guides/publish.md?plain=1#L36) to add screenshots or do you think there are more that would be helpful?

Reply:
D'oh, we can! 😄